src/main/scala/com/gd/twitterstreaming/StreamingTweetsJob.scala

```diff
@@ -0,0 +1,40 @@
+package com.gd.twitterstreaming
+
+import org.apache.log4j.Logger
+import org.apache.spark.SparkConf
+import org.apache.spark.streaming.{Seconds, StreamingContext}
+
+object StreamingTweetsJob {
+
+  val log = Logger.getLogger(StreamingTweetsJob.getClass.getName)
+
+  def main(args: Array[String]): Unit = {
+
+    val ssc = setSparkSessionConf
```

Comment:
To my mind, there is no reason to start the context if Application configuration is invalid.

Reply:
Fixed.

Reply:
Issue fixed in scope of 
https://github.com/ipetrushin-gd/data-engineering-practice/pull/2